### DISCUSSION TITLE
German Varia RCT716 - some questions

### DISCUSSION THREAD

**Discussion by DohbiK** [maca88/SmartBikeLights#45] [General]
Hi maca88,

first of all - thanks a lot for your amazing development!
I really like it!
But I have some questions:
First:
Would it be possible to include the Varia RCT716 (that is the German version of the RCT715 - without flash mode) into the support?
And second:
Is there perhaps a way to "teach" the German RCT716 to flash like its international brother the RCT715?

best regards,
Tobias

**Comment by maca88** [comment]
Hi,

> Would it be possible to include the Varia RCT716 (that is the German version of the RCT715 - without flash mode) into the support?

Varia RCT716 is now available in the configurator,

> Is there perhaps a way to "teach" the German RCT716 to flash like its international brother the RCT715?

Unfortunately, there is no way to change RCT716 to behave as RCT715 (as far as I know).
